pyincore/analyses/housingvaluationrecovery/housingvaluationrecovery.py

```diff
@@ -51,53 +53,56 @@ def get_spec(self):
                 {
```

Comment:
I think we need to focus on adding the description here so that it is read by the tools.

Reply:
I mean the analysis description in the spec's top level description field

Reply:
Sorry, I don't think that I understand what you mean.

Reply:
Updated